### PR TITLE
Nav Management Drawer

### PR DESCRIPTION
**Pull request by onnovisser**
### Description

Completes the Nav Management drawer, with nav overview card and token price change.
Also fixes some data and data not being refetched in a couple places.

#2045

### Approvals

- [ ] Dev

### Screenshots

image

### CODE REVIEW

File: centrifuge-app/src/utils/useLiquidity.tsx [hunk 1]
```diff
@@ -12,28 +12,25 @@ type Liquidity = {
...
-    ],
+    ['solution', poolId],
     (cent) => cent.pools.submitSolution([poolId], { dryRun: true }),
     {
-      enabled: !!poolId && !!poolOrders,
+      enabled: !!poolId,
     }
   )
 
+  console.log('solution', solution)
```
Comment (sophialittlejohn) [inline centrifuge-app/src/utils/useLiquidity.tsx:32]: oops

**Review by sophialittlejohn — APPROVED** [review]
LGTM 💯

File: centrifuge-app/src/pages/NavManagement/NavManagementAssetTable.tsx [hunk 2]
```diff
@@ -73,46 +81,55 @@ export function NavManagementAssetTable({ poolId }: { poolId: string }) {
...
       formIndex: -1,
       maturity: '',
       oldValue: '',
     },
   ]
 
+  const { ordersFullyExecutable } = useLiquidity(poolId)
+
   const { execute, isLoading } = useCentrifugeTransaction(
     'Set oracle prices',
```
Comment (hieronx) [inline centrifuge-app/src/pages/NavManagement/NavManagementAssetTable.tsx:94]:
```suggestion
    'Update NAV',
```

**Comment by hieronx** [comment]
Just did a quick test and this looks great! 🔥 Left 1 nit above

**Pull request merged**